### PR TITLE
Added map and secured SQL commands

### PR DESCRIPTION
**Pull request by buuck**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -296,84 +338,54 @@ def calculate_biodiversity_score(start_year, end_year, project_name):
...
+    return scores
 
 def motherduck_list_projects(author_id):
-    return con.sql(
-        f"""
-        SELECT DISTINCT name FROM project WHERE authorId = '{author_id}'
-    """
-    ).df()
+    return \
+        con.execute("SELECT DISTINCT name FROM project WHERE authorId = ? AND geometry != 'null'", [author_id]).df()
```
Comment (buuck) [inline app.py:357]: At the moment the DB has null values set to the string 'null' instead of the null character. We should try to avoid this if possible.

**Comment by buuck** [comment]
Probably we actually want to use this for the map, especially since it will play better with samgeo https://github.com/opengeos/leafmap/issues/362

File: app.py (hunk 1 above)
Comment (pgzmnk) [reply app.py:357]: Great call. Added to the docket.

Comment (buuck) [reply app.py:357]: Is this the notion? I am having trouble finding it in my email or on Slack.

**Pull request merged**